### DISCUSSION TITLE
Blocked Host - error

### DISCUSSION THREAD

**Discussion by r4nchy** [Freika/dawarich#493] [Q&A] [answered]
I am getting this error when access the url in browser, what am i doing wrong ??

![](https://i.imgur.com/YtXwaHm.png)

**Comment by Freika** [comment, marked as answer]
https://dawarich.app/docs/environment-variables-and-settings you're looking for `APPLICATION_HOSTS` (plural)

**Reply by r4nchy** [reply]
thank you it works now

```
APPLICATION_HOSTS: localhost,dawarich.example.org
```
